### ISSUE TITLE
[BUG]conduit build fail with not found CodeAnalyzer

### ISSUE THREAD

**Issue by delphi007** [conduit-dart/conduit#198]
**Describe the bug**
conduit build fail

**To Reproduce**
1. cd <prjdir>
2. dart pub get
3. conduit build
-- Conduit CLI Version: 4.3.9
-- Conduit project version: 4.3.8
Starting build process...
Resolving ASTs...
Generating runtime...
Generated runtime at 'file:///tmp/build/generated_runtime/'.
Compiling package 'conduit_runtime'...
Package 'conduit_runtime' compiled to 'file:///tmp/build/packages/conduit_runtime/'.
Compiling package 'conduit_core'...
Package 'conduit_core' compiled to 'file:///tmp/build/packages/conduit_core/'.
Compiling package 'conduit_config'...
Package 'conduit_config' compiled to 'file:///tmp/build/packages/conduit_config/'.
Copying application package (from 'file:///Users/delphi/workdir/vda_auto_server/')...
Application packaged copied to 'file:///tmp/build/packages/vda_auto_server/'.
Fetching dependencies (--offline --no-precompile)...
Finished fetching dependencies.
Compiling...
*** Uncaught error
    Bad state: Bad state: 'dart2native' failed with the following message: Info: Compiling with sound null safety.
/Users/delphi/.pub-cache/hosted/pub.flutter-io.cn/conduit-4.3.9/lib/src/migration_source.dart:21:22: Error: Method not found: 'CodeAnalyzer'.
    final analyzer = CodeAnalyzer(uri);
Error: AOT compilation failed
Generating AOT kernel dill failed!
#0      generateNative (package:dart2native/generate.dart:72:7)
<asynchronous suspension>
#1      CompileNativeCommand.run (package:dartdev/src/commands/compile.dart:312:7)
<asynchronous suspension>
#2      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
<asynchronous suspension>
#3      DartdevRunner.runCommand (package:dartdev/dartdev.dart:241:18)
<asynchronous suspension>
#4      runDartdev (package:dartdev/dartdev.dart:65:18)
<asynchronous suspension>
#5      main (file:///opt/s/w/ir/x/w/sdk/pkg/dartdev/bin/dartdev.dart:11:3)
<asynchronous suspension>

  **** Stacktrace
  * #0      Build.compile (package:conduit_runtime/src/build.dart:161:7)
  * <asynchronous suspension>
  * #1      Build.execute (package:conduit_runtime/src/build.dart:121:7)
  * <asynchronous suspension>
  * #2      BuildExecutable.execute (<data:application/dart>:13:240)
  * <asynchronous suspension>
  * #3      main (<data:application/dart>:9:18)
  * <asynchronous suspension>
  ****

**Expected behavior**
build success

**Analysis**
Actually this is not what my project error but conduit. Have checked pub_cache, the conduit_runtime/4.3.7 exist which include the CodeAnalyzer class definition.

**System Information:**
 - OS:  build fail on both Mac(intel) and Ubuntu
 - Dart Version: 2.19.3
 - Conduit Version: 4.3.9

**Comment by pst9354** [comment]
I have the same problem here.
Using conduit 4.3.6
Dart 2.19.4

**Comment by j4qfrost** [comment]
@delphi007 @pst9354 can you both try purging all the conduit packages from pub cache and try rerunning conduit build?

**Comment by j4qfrost** [comment]
This might be an issue "hosted/pub.flutter-io.cn" @pst9354 what location are you using for your cache?

**Comment by delphi007** [comment]
@j4qfrost After 'rm -rf conduit*' in hosted/pub.flutter-io.cn and 'dart pub get' , I got the same error from 'conduit build'. Then I switched the pub repository back to pub.dev, remove 'conduit*', rerun 'dart pub get & conduit build', the same error.

Btw, the version of each conduit package is in the picture. From the 'conduit build' output , Conduit Cli is 4.3.9, Conduit Project version is 4.3.8, although I specified the project the conduit plugin version 4.3.9 in pubspec.yaml. Just FYI.
<img width="254" alt="image" src="https://user-images.githubusercontent.com/11587194/226061999-1aa4aa8b-fe12-433f-a29d-20d2adb4da61.png">
